pattern_generator_test.rb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Haha, because I get excited and push when I get the code right and forget about the bottom of the file!! I'll be better on that:)